arci-ros/tests/util/msg_helper.rs

```diff
@@ -0,0 +1,27 @@
+#![cfg(target_os = "linux")]
+
+use std::sync::mpsc;
+
+/// # subscribe ROS message helper
+///
+/// using for inspect specific massage type.
+/// Message is displayed on screen and sent to ``mpsc receiver``
+///
+/// # Panic!
+///
+/// If subscriber can't be construct, this function is panic.
+///
+#[allow(dead_code)]
```

Comment:
Is this needed?

Reply:
If this code is not, cllipy says warning.
But be not called is also exist because using test helper.

Reply:
Certainly.
However, if you write the test code of this function in this file, this attribute might not be required.